flatdata-generator/flatdata/generator/templates/rust/archive.jinja2

```diff
@@ -109,15 +109,13 @@ impl ::std::fmt::Debug for {{archive.name}} {
 }
 
 impl {{archive.name}} {
+    #[allow(unused_imports, unused_variables)]
     pub fn open(storage: flatdata::StorageHandle)
         -> ::std::result::Result<Self, flatdata::ResourceStorageError>
     {
-        #[allow(unused_imports)]
         use flatdata::SliceExt;
```

Comment:
I remember these were added to make sure the "empty" archive does not generate errors (as it would not contain any code in here). Can you double-check that that works fine?